### ISSUE TITLE
Some fonts installed on my Windows machine are not seen by Musescore

### ISSUE THREAD

**Issue by mike-spa** [musescore/MuseScore#16188]
**Describe the bug**
Some _**text**_ fonts installed on my Windows machine are not seen by Musescore. This isn't a new problem, it's been reported several times in the past, see:
https://musescore.org/en/node/280244
https://musescore.org/en/node/282501
https://musescore.org/en/node/300756
However, it seems that the solution in that case was to select on Windows "Install for all users".  
In the example I show below, however, this still doesn't work. In general, I've noticed that the list of fonts in Musescore shows less fonts than what I see in other apps. I have no clue if this is a Qt-problem, a Windows-problem, or is machine-specific. It will definitely need some more testing.

@Tantacrul @oktophonie Not sure how to prioritize this

https://user-images.githubusercontent.com/93707756/216295180-8a04bc47-1ad4-4d09-bba0-abcb924dfa64.mp4

(By the way, here is the font I was trying in this example https://github.com/benwiggy/nepomuk)

**Comment by Jojo-Schmitz** [comment]
That issue https://musescore.org/en/node/280244 had been fixed in 9866ce0dbf, for 3.5.0

**Comment by Jojo-Schmitz** [comment]
So this relates to #16189

**Comment by jonarnoldmusic** [comment]
In case it's helpful, a specific font that works in Musescore 3 and not Musescore 4 for me is the Nepomuk font (available for free or donation at https://www.notationcentral.com/product/nepomuk/). In addition, I cannot see Minion Pro (a commonly used lyric font) in Musescore (either 3 or 4) when installed from Adobe Creative Cloud.

**Comment by jonarnoldmusic** [comment]
> The small caps variants of that font may be useful to test with, as they do seem to register although the naming is not consistent: https://www.scoringnotes.com/news/the-nepomuk-text-font-it-takes-a-certain-type/#comment-35627

To be clear, if I only install the regular font variant, Nepomuk is not visible in MS4 as a font. If I install the small caps variant, Nepomuk and Nepomuk SC appear as fonts, but they both seem to be small caps.

In this picture, the title is in Nepomuk and the subtitle is in Nepomuk SC:
![Screenshot 2024-09-01 222142](https://github.com/user-attachments/assets/f1352b0c-1271-45f9-9574-a1cc2da6d3b5)

**Comment by SimusMusicus** [comment]
In my case (Musescore 4.4.4 / Windows 10), adding the option :
--platform windows:fontengine=directwrite
after ‘C:\Program Files\MuseScore 4\bin\MuseScore4.exe’ in the properties of the Musescore shortcut on the desktop fixed the problem!
This workaround also allowed me to use other fonts (Papyrus, Pieces of Eight, Dutch and Harley...) that didn't appear in the font drop-down list.

**Comment by SilverGreen93** [comment]
I have the same problem in MuseScore 4.5.2. My `Red Hat Text` font is not listed in the fonts dropdown.
If I start with the parameters mentoined above `--platform windows:fontengine=directwrite`, the font is listed, but all fonts are displayed in a weird way (kind-of pixelated or non anti-aliased / not smooth)

**Comment by cbjeukendrup** [comment]
Yes, as soon as anyone finds a workaround for that problem, we'll make directwrite the default, but unfortunately that's indeed a blocker so far.

**Comment by Jojo-Schmitz** [comment]
See also the workaround mentioned in https://musescore.org/en/node/377949#comment-1319953
Which is turning an OTF into a TTF
